db/payment_icons.yml

```diff
@@ -2210,3 +2210,11 @@
   name: qliro
   label: Qliro
   group: other
+-
+  name: overstockciticobrand
```

Comment:
Any reason why you are using different name than the label?

Reply:
Yes, just want to follow a naming convention companyNameBankNameCardName, to make it easy for any future additions or updates..
Label is how we refer those cards across our website.